### PR TITLE
Remove the query builder and add a query help dialog.

### PR DESCRIPTION
**Pull request by jrobbins**
The query builder was a nice idea, but it would never have been able to express all query terms in widgets without becoming very complex.   Instead, we will try simply having the user use the search box, but with support in the form of a help dialog (and autocomplete in the future).

In this PR:
* Add some shared CSS inspired by recent work on the other dashboard.
* Delete all the query builder configuration and logic from chromedash-feature-filter.
* Replace the `tune` icon with a `help` icon and make it open a help dialog.
* Implement a help dialog with some examples, explanation of syntax, and a list of available fields.
* Refactor the query builder configuration constants into a new file that is used to populate the help dialog (and autocomplete in the future).

### CODE REVIEW

**Comment by jrobbins** [comment]
Screenshot of the open help dialog:
image

File: client-src/elements/chromedash-search-help-dialog.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    return html`
+      <h3>Search syntax</h3>
+      <section>
+
+      <p>A search query consists of a series of terms that are separated by
+      spaces. Terms can be single words or conditions.</p>
+
+      <p>When searching for words, the results will include features that
+      include those words in any field of the feature entry.  We do not support
+      searching for wildcards, partial words, puctuation, or quoted phrases.</p>
```
Comment (KyleJu) [inline client-src/elements/chromedash-search-help-dialog.js:101]:
```suggestion
      searching for wildcards, partial words, punctuation, or quoted phrases.</p>
```

File: client-src/elements/chromedash-search-help-dialog.js [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+      Search terms are implicitly joined together by a logical-AND.
+      However, you may type the keyword <code>OR</code> between terms as
+      a logical-OR operator.  We do not support parenthesis in queries.
+      </p>
+
+      </section>
+    `;
+  }
+
+  renderFieldRow(qf) {
```
Comment (KyleJu) [inline client-src/elements/chromedash-search-help-dialog.js:124]:
```suggestion
  renderFieldRow(queryField) {
```

**Review by KyleJu — APPROVED** [review]
Nice change. I like the idea!

File: client-src/elements/chromedash-search-help-dialog.js (hunk 2 above)
Comment (KyleJu) [reply client-src/elements/chromedash-search-help-dialog.js:124]: and ditto below

**Pull request merged**